PHPCI/Model/Base/ProjectBase.php

```diff
@@ -148,6 +151,9 @@ class ProjectBase extends Model
             'type' => 'int',
             'length' => 11,
         ),
+        'archive' => array(
+            'type' => 'boolean',
```

Comment:
Did you modify this file by hand, or did you use ./console db:generate to update it? I'm intrigued that it used a "type" value of boolean.